LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2021 by David D. Riddle and the Board of Trustees of the
```

Comment:
Do we want a specific name here? 
I thought the general guidance was to keep it more generic and only refer to UIUC, the Board of Trustees, etc.

Reply:
This is what I learned in SDG from Jon Roma. It is also just a straight copy from awscli-login.

Reply:
This is fine. Thanks.